bin/lint-schema-ts-file.sh

```diff
@@ -0,0 +1,14 @@
+# Validates whether generated schema.ts file is up to date with actual GraphQL API
+# Renames original schema.ts file, generates a new one, compares and rolls changes back
+
+mv apps/condo/schema.ts apps/condo/schema.temp.ts
+yarn workspace @app/condo maketypes
```

Comment:
to this cript you need to prepare .env. `cp .env.example .env` can help. 
Probably, it's better to move it inside the nodejs.condo.test.yml because it's required ENV